packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -220,7 +187,12 @@ function nodeDataToMapTree(
 	const flexNode = tryGetFlexNode(data);
 	if (flexNode !== undefined) {
 		if (isMapTreeNode(flexNode)) {
-			return flexNode.mapTree;
+			// TODO: mapTreeFromNodeData modifies the trees it gets to add defaults.
```

Comment:
I don't think this really causes any really issues, so its just debt relevant to people working on this code: thus I don't think we need an item tracking it, especially since  we now do have some bugs tracking the actual correctness issues in this code which will likely result in refactorings that also address this.